includes/html/functions.inc.php

```diff
@@ -1610,92 +1610,57 @@ function get_device_name($device)
  * Returns state generic label from value with optional text
  */
 
-function get_state_label($state_value, $state_text_param = null)
+function get_state_label($sensor)
 {
-    switch ($state_value) {
+    $state_translation = array();
+    $state_translation = dbFetchRows('SELECT * FROM state_translations as ST, sensors_to_state_indexes as SSI WHERE ST.state_index_id=SSI.state_index_id AND SSI.sensor_id = ? AND ST.state_value = ? ', array($sensor['sensor_id'], $sensor['sensor_current']));
```

Comment:
You probably want dbFetchRow() here, that way you don't have to do the silly [0] stuff below

Reply:
Updated that, thanks for the tip!